### PR TITLE
Fix tag calculation

### PR DESCRIPTION
**Pull request by sigurdm**
Two issues:
primary library was calculated wrong
flutter-web should allow dart:io imports

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.13.1+3
+
+* Allow import of dart:io on platform web when calculating tags.
```
Comment (jonasfj) [inline CHANGELOG.md]: Let's special case `package:flutter` instead, importing `dart:io` won't work, see:
https://flutter.dev/docs/development/platform-integration/web#can-i-use-dartio-with-a-web-app

File: lib/src/tag_detection.dart [hunk 2]
```diff
@@ -224,6 +228,9 @@ class _PackageGraph implements _DirectedGraph<String> {
   Set<String> directSuccessors(String packageDir) {
     final pubspec = _pubspecCache.pubspecOfPackage(packageDir);
     return pubspec.dependencies.keys
+        // HACK: Ignore package:flutter as it is provided by the SDK.
+        // TODO(jonasfj): Look at the package dependency kind, we should ignore SDK dependencies
+        .filter((name) => name == 'flutter')
```
Comment (jonasfj) [inline lib/src/tag_detection.dart]: @sigurdm, does this seem right?

File: lib/src/tag_detection.dart [hunk 3]
```diff
@@ -153,6 +153,10 @@ class LibraryGraph implements _DirectedGraph<Uri> {
       if (uriString.startsWith('dart:') || uriString.startsWith('dart-ext:')) {
         return <Uri>{};
       }
+      // HACK: Package flutter comes from the SDK, we do not want to look at its import graph.
+      if (uriString.startsWith('package:flutter/') {
+        return <Uri>{};
```
Comment (jonasfj) [inline lib/src/tag_detection.dart]: @sigurdm, does this seem right?

File: lib/src/tag_detection.dart (hunk 2 above)
Comment (sigurdm) [reply lib/src/tag_detection.dart]: Why is this needed?

File: CHANGELOG.md (hunk 1 above)
Comment (jonasfj) [inline CHANGELOG.md]: update changelog

**Pull request merged**